File: 2.0/python/texts/findTextBroadcasts.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.texts.findTextBroadcasts(
    # filter by name
    name='broadcast name',
    # filter by label
    label='my label',
    # filter only broadcasts in RUNNING state
    running=True,
    # search offset
    offset=0,
    # return 10 items per request
    limit=10,
    # return only specific fields
    fields='items(id,name,status,message)'
).result()

print(response)

File: 2.0/php/texts/findTextBroadcasts.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->findTextBroadcasts();
        $request->getOperationConfig()->setQueryParameters(array("name" => "broadcast name",
                                                                 "label" => "my label",
                                                                 "running" => true,
                                                                 "offset" => 0,
                                                                 "limit" => 10,
                                                                 "fields" => "items(id,name,status,message)"));
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();
